plasmapy/formulary/collisions.py

```diff
@@ -2297,9 +2345,156 @@ def Lorentz_collision_frequency(self):
             * (self.field_particle.charge_number * e.esu) ** 2
             * self.Coulomb_log
             * self.n_b
-            / (self.test_particle.mass**2 * self.v_a**3)
+            / (self.test_particle.mass**2 * self._v_a**3)
         ).to(u.Hz)
 
+    @cached_property
+    def _mean_thermal_velocity(self):
+        """
+        Parameter used in enforcing the definition of "slowly flowing" Maxwellian
+        particles. See Callen (Eq. 2.113).
+        """
+
+        v_T_b = thermal_speed(self.T_b, self.field_particle)
+
+        return (self.v_a**2 + v_T_b**2) ** 0.5
+
+    @cached_property
+    def _is_slowly_flowing(self):
+        """
+        Criteria used in determining whether `Maxwellian_avg_ei_collision_freq` and
+        `Maxwellian_avg_ii_collision_freq` can be applied to the specified species.
+        """
+
+        return self.v_a / self._mean_thermal_velocity < 0.1
+
+    @cached_property
+    def Maxwellian_avg_ei_collision_freq(self):
+        r"""Average momentum relaxation rate for a slowly flowing Maxwellian
+        distribution of electrons, relative to a population of stationary ions.
+
+        In order to use this attribute, ``test_particle`` must be an electron
+        and ``field_particle`` must be an ion.
+
+        This function assumes that both populations are Maxwellian, and :math:`T_{i} \lesssim T_{e}`
+
+        :cite:t:`callen:unpublished` provides a derivation of this as an
+        average collision frequency between electrons and ions for a
+        Maxwellian distribution. It is thus a special case of the collision
+        frequency with an averaging factor, and is on many occasions
+        in transport theory the most relevant collision frequency that has
+        to be considered. It is heavily related to diffusion and resistivity
+        in plasmas.
+
+        Raises
+        ------
+        `~plasmapy.utils.exceptions.PhysicsError`
+            The specified species aren't slowly flowing
+
+        `ValueError`
+            If the specified interaction isn't electron-ion, or ``n_a`` isn't specified.
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> n_a = 1e26 * u.m**-3
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e26 * u.m**-3
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> electron_ion_collisions = CollisionFrequencies(
+        ...     "e-", "Na+", n_a=n_a, T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> electron_ion_collisions.Maxwellian_avg_ei_collision_freq
+        <Quantity 2906316911556553.5 Hz>
+        """
+
+        if self.v_a is None:
```

Comment:
Personal opinion - it gets annoying when a function tells you it needs something, you add it, and then a new error pops up. Instead, I would check for each of these, add the error messages to a list if the conditions fail, and then if the list is not empty raise an error and print the whole list.

Reply:
Good idea! I was able to kill two birds with one stone by using the new `_validate_arguments` decorator, it'll give you a list of the parameters that you didn't specify and also removes the need for a lot of the `ValueError` if statements